.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: 💬 Questions & Help
+    url: https://github.com/cisagov/.github/blob/develop/QUESTIONS.md
```

Comment:
This might be a good place to link to this repo's Wiki instead of the generic page:

https://github.com/cisagov/cyhy_amis/wiki

Reply:
Unless you two have added useful content, that wiki does not hold a lot of value.

Reply:
I could add a link specifically to [cisagov/cyhy-system](https://github.com/cisagov/cyhy-system), but I don't think there's much project-specific here to work with right now.

Reply:
I added an entry for the [cisagov/cyhy-system](https://github.com/cisagov/cyhy-system) Issues page in 948d24f to encourage issue creation for the CyHy environment to happen there instead. Please let me know any thoughts you all might have.

Reply:
I'd change the wording to be directive, e.g.:

`Create issues in the linked cyhy-system repository except for infrastructure issues with the AMIs, which should be created here.`